public/app/plugins/panel/geomap/layers/data/circlesOverlay.ts

```diff
@@ -0,0 +1,180 @@
+import { MapLayerRegistryItem, MapLayerConfig, MapLayerHandler, PanelData, GrafanaTheme2 } from '@grafana/data';
+import { dataFrameToLocations } from './utils'
+import { FieldMappingOptions } from '../../types'
+import Map from 'ol/Map';
+import Feature from 'ol/Feature';
+import * as layer from 'ol/layer';
+import * as source from 'ol/source';
+import * as style from 'ol/style';
+import {Point} from 'ol/geom';
+import { fromLonLat } from 'ol/proj';
+import tinycolor from 'tinycolor2';
+
+export interface CirlceConfig {
+  fieldMapping: FieldMappingOptions,
+  minSize: number,
+  maxSize: number,
+  opacity: number,
+}
+
+const defaultOptions: CirlceConfig = {
```

Comment:
typo?

Reply:
Yep. Thank you for catching this :sweat_smile: